vault/core.go

```diff
@@ -2632,6 +2637,84 @@ func (c *Core) SetConfig(conf *server.Config) {
 	c.logger.Debug("set config", "sanitized config", string(bz))
 }
 
+func (c *Core) getCustomResponseHeaders(la string) []*ListenerCustomHeaders {
+	if c.customListenerHeader == nil {
+		c.logger.Debug("failed to find the custom response headers configuration")
```

Comment:
Looks like this debug message as well as the warn below would be emitted on every request, probably not what we want.